### PR TITLE
Removing some authorizations for departed users [skip ci]

### PR DESCRIPTION
**Pull request by hyperbolic2346**
Pour one out for the departed 🍺

I have a feeling more should be removed from here.

### CODE REVIEW

**Review by revans2 — DISMISSED** [review]

File: .github/workflows/blossom-ci.yml [hunk 1]
```diff
@@ -35,11 +35,9 @@ jobs:
     # This job only runs for pull request comments
     if: contains( '\
       abellina,\
-      andygrove,\
       anfeng,\
       firestarman,\
       GaryShen2008,\
-      jbrennan333, \
       jlowe,\
       krajendrannv,\
```
Comment (sameerz) [inline .github/workflows/blossom-ci.yml:42]: Please remove krajendrannv as well.  
```suggestion
```

**Review by jlowe — APPROVED** [review]

**Comment by jlowe** [comment]
@hyperbolic2346 looks like [signoff](https://github.com/NVIDIA/spark-rapids/blob/branch-24.06/CONTRIBUTING.md#sign-your-work) is missing on the commits.

**Review by sameerz — APPROVED** [review]

**Review by NvTimLiu — APPROVED** [review]
LGTM,

Can you also help to update in spark-rapids-jni? Thanks!

https://github.com/NVIDIA/spark-rapids-jni/blob/branch-24.06/.github/workflows/blossom-ci.yml

**Comment by sameerz** [comment]
> LGTM,
> 
> Can you also help to update in spark-rapids-jni? Thanks!
> 
> https://github.com/NVIDIA/spark-rapids-jni/blob/branch-24.06/.github/workflows/blossom-ci.yml

https://github.com/NVIDIA/spark-rapids-jni/pull/1981

**Comment by sameerz** [comment]
build

**Pull request merged**